styles/index.scss

```diff
@@ -2,12 +2,28 @@
 @use './colors';
 @use './sections/nav';
 @use './sections/hero';
+@use './sections/mission';
 
 body {
   background-color: colors.$color-blue;
   color: colors.$color-white;
   container: body / inline-size;
   font-family: 'Tomorrow', sans-serif;
+  font-weight: 300;
+  font-size: 1rem; // 16px
+  line-height: 200%;
+  letter-spacing: 0.05rem;
+  position: relative;
+  height: 100%;
+
+  @container body (min-width: 1024px) {
+    text-align: center;
+  }
+
+  mark {
+    font-weight: 500;
+    display: inline-block;
+  }
```

Comment:
To make this long file of styles easier to understand, it might make more sense to group like things together and move these styles next to the `.mark-teal` class styles on line 205?